packages/react/src/components/DetailsList/DetailsRow.base.tsx

```diff
@@ -292,9 +299,11 @@ export class DetailsRowBase extends React.Component<IDetailsRowBaseProps, IDetai
         {showCheckbox && (
           <div role="gridcell" aria-colindex={1} data-selection-toggle={true} className={this._classNames.checkCell}>
             {onRenderCheck({
+              id: this._checkboxId,
               selected: isSelected,
               anySelected: isSelectionModal,
               'aria-label': checkButtonAriaLabel,
```

Comment:
Can an element have *both* an `aria-label` and an `aria-labelledby` value? My experience so far with screen-readers is that the don't react too well to this. It works okay with an `aria-labelledby` and an `aria-describedby`, though.

Reply:
Hmm my understanding is that aria-labelledby will simply take priority over aria-label but I'm not too sure of the side-effects of it - perhaps @smhigley can shed some more light to this

Reply:
Yup! Aria-labelledby allows you to use a self-referencing `id`, and in the name computation `aria-labelledby` will never be followed twice, whether self-referencing or otherwise. Using both aria-label + aria-labelledby allows you to compose a label based on both the element's own text, and another element's text. So, for example, the following button:

```
<button aria-label="select" id="id1" aria-labelledby="id1 id2">(icon)</button>
<div id="id2">some item</div>
```

Would have a computed name of "select some item".